src/test/java/com/commercetools/sync/products/helpers/variantreferenceresolver/WithCustomerReferencesTest.java

```diff
@@ -0,0 +1,109 @@
+package com.commercetools.sync.products.helpers.variantreferenceresolver;
+
+import com.commercetools.sync.products.ProductSyncOptions;
+import com.commercetools.sync.products.ProductSyncOptionsBuilder;
+import com.commercetools.sync.products.helpers.VariantReferenceResolver;
+import com.commercetools.sync.services.CategoryService;
+import com.commercetools.sync.services.ChannelService;
+import com.commercetools.sync.services.CustomerGroupService;
+import com.commercetools.sync.services.ProductService;
+import com.commercetools.sync.services.TypeService;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.products.ProductVariantDraft;
+import io.sphere.sdk.products.ProductVariantDraftBuilder;
+import io.sphere.sdk.products.attributes.AttributeDraft;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static com.commercetools.sync.products.ProductSyncMockUtils.createReferenceObject;
+import static com.commercetools.sync.products.ProductSyncMockUtils.getReferenceSetAttributeDraft;
+import static com.commercetools.sync.products.helpers.VariantReferenceResolver.REFERENCE_ID_FIELD;
+import static com.commercetools.sync.products.helpers.VariantReferenceResolver.REFERENCE_TYPE_ID_FIELD;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class WithCustomerReferencesTest {
+    private VariantReferenceResolver referenceResolver;
+
+    /**
```

Comment:
IMO this code is easy to read, I would remove all these comments `Sets up the services....` on all tests of that PR. So we will not manage comments if sth changed,  WDYT ?

Reply:
2289b34